src/panels/lovelace/components/hui-theme-select-editor.ts

```diff
@@ -17,6 +18,18 @@ export class HuiThemeSelectionEditor extends hassLocalizeLitMixin(LitElement) {
     };
   }
 
+  protected constructor() {
+    super();
+    this._initializing = true;
+  }
+
+  protected async firstUpdated(): Promise<void> {
+    if (this._initializing) {
+      await this.updateComplete;
```

Comment:
Bit surprised about this, isn't this method called after the first time has been rendered, so awaiting this promise means you wait for 2nd render?